lib/kinematics/test/rotation3_test.cpp

```diff
@@ -8,26 +8,51 @@
 using kinematics::Matrix3d;
 using kinematics::Rotation3;
 using kinematics::Vector4d;
+using kinematics::Vector3d;
 
 TEST(Rotation3Test, TestNullPtr) {
-  Rotation3 Rx;
-  Vector4d quat;
-  quat << 1.0, 0.0, 0.0, 0.0;
-  EXPECT_TRUE(Rotation3::GetRotation3FromQuaternion(quat, &Rx));
-  EXPECT_FALSE(Rotation3::GetRotation3FromQuaternion(quat, nullptr));
+    Rotation3 Rx;
+    Vector4d quat;
+    quat << 1.0, 0.0, 0.0, 0.0;
+    EXPECT_TRUE(Rotation3::GetRotation3FromQuaternion(quat, &Rx));
+    EXPECT_FALSE(Rotation3::GetRotation3FromQuaternion(quat, nullptr));
 }
 
 TEST(Rotation3Test, TestQuaternion) {
-  // A ground truth from http://www.energid.com/resources/quaternion-calculator/
-  Vector4d quat;
-  Matrix3d Rt;
-  quat << 0.18260418, 0.36510837, -0.54771256, 0.73031674;
-  Rt << -0.666703178340727, -0.666666666666667, 0.333260299984754,
-        -0.133231088645414, -0.333333333333334, -0.933347934002774,
-        0.733318722663434, -0.666666666666667, 0.133413667016633;
-
-  Rotation3 Rx;
-  Rx.FromQuaternion(quat);
-  Matrix3d Rm = Rx.GetRotationMatrix();
-  EXPECT_TRUE((Rm - Rt).norm() < 1e-6);
+    // A ground truth from http://www.energid.com/resources/quaternion-calculator/
+    Vector4d quat;
+    Matrix3d Rt;
+    quat << 0.18260418, 0.36510837, -0.54771256, 0.73031674;
+    Rt << -0.666703178340727, -0.666666666666667, 0.333260299984754,
+            -0.133231088645414, -0.333333333333334, -0.933347934002774,
+            0.733318722663434, -0.666666666666667, 0.133413667016633;
+
+    Rotation3 Rx;
+    Rx.FromQuaternion(quat);
+    Matrix3d Rm = Rx.GetRotationMatrix();
+    EXPECT_TRUE((Rm - Rt).norm() < 1e-6);
+}
+
+TEST(Rotation3Test, TestRotation) {
+    // A ground truth from Octave (https://www.gnu.org/software/octave/)
+    Vector3d v3dSrc;
+    Vector3d v3dTgt, v3dResult;
+    Vector3d v3dInvTgt, v3dInvResult;
```

Comment:
Name the variables using "snake" style, like v3d_src, v3d_tgt...
On there other hand, we use "Camel" style to name classes, functions, matrices or global constants like
Rotation3Test, InverseRotate()...